src/command_modules/azure-cli-backup/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+1.0.5
++++++
+* Bugfix: Supplying non-existent name or natively formatted name for container causes stack trace.
```

Comment:
This looks like the issue name, what was the fix and to which command(s)?  The see the fix for if a non-existent name is supplied... does it actually support using the natively formatted name?

Reply:
@tjprescott, good catch! I'm partially fixing the bug: supplying non-existent name would not cause an issue now. However, still not handling natively formatted name. Will take it up in the next release. Will update the release history to reflect the same.